CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.15.1
+
+* Add the machine.Shutdown() method, enabling access to the SendCtrlAltDel API
```

Comment:
In other projects we prefix entries with "Feature", "Enhancement", or "Bugfix".  How do you feel about including one of those here?

Reply:
My feeling can best be summarized as "meh". The distinction can really be somewhat subjective anyway, and I don't think it contributes a whole lot.